src/main/resources/application.yml

```
@@ -67,6 +73,9 @@ spring:
     replica-set-name: null
 
 application:
+  transportDataUrl: ${TRANSPORT_URL:http://localhost:5050/agreements/v1/adapter/transmodel/export}
+  organizationKey: ${ORGANIZATION_KEY:Et-Client-Name}
+  organizationValue: ${ORGANIZATION_VALUE:Digdir}
```

Trenger ikke være miljøvariabler for test, og bruk tilfeldige testverdier
```suggestion
  transportDataUrl: http://localhost:5050/agreements/v1/adapter/transmodel/export
  organizationKey: Header-Key
  organizationValue: test
```